armtemplates/template.bicep

```diff
@@ -0,0 +1,280 @@
+      addressPrefixes: [
+        '10.0.0.0/16'
+      ]
+    }
+    subnets: [
+      {
+        name: 'default'
+        //id: virtualNetworks_iagora_vnet_name_default.id
+        properties: {
+          addressPrefix: '10.0.0.0/24'
+          delegations: []
+          privateEndpointNetworkPolicies: 'Disabled'
+          privateLinkServiceNetworkPolicies: 'Enabled'
+        }
+        //type: 'Microsoft.Network/virtualNetworks/subnets'
+      }
+    ]
+    virtualNetworkPeerings: []
+    enableDdosProtection: false
+  }
+}
+
+// VIRTUAL MACHINE
+
+resource virtualMachines_iagora_name_resource 'Microsoft.Compute/virtualMachines@2024-03-01' = {
+  name: virtualMachines_iagora_name
+  location: 'francecentral'
+  zones: [
+    '1'
+  ]
+  properties: {
+    hardwareProfile: {
+      vmSize: vmSize
+    }
+    additionalCapabilities: {
+      hibernationEnabled: false
+    }
+    storageProfile: {
+      imageReference: {
+        publisher: 'canonical'
+        offer: '0001-com-ubuntu-server-focal'
+        sku: '20_04-lts-gen2'
+        version: 'latest'
+      }
+      osDisk: {
+        osType: 'Linux'
+        name: '${virtualMachines_iagora_name}_OsDisk_1_bcb7e14b83d24fc7a077197bdf465401'
+        createOption: 'FromImage'
+        caching: 'ReadWrite'
+        managedDisk: {
+          storageAccountType: 'Premium_LRS'
+          /*id: resourceId(
+            'Microsoft.Compute/disks',
+            '${virtualMachines_iagora_name}_OsDisk_1_bcb7e14b83d24fc7a077197bdf465401'
+          )*/
+        }
+        deleteOption: 'Delete'
+        diskSizeGB: 30
+      }
+      dataDisks: []
+      diskControllerType: 'SCSI'
+    }
+    osProfile: {
+      computerName: virtualMachines_iagora_name
+      adminUsername: virtualMachines_iagora_name
+      linuxConfiguration: {
+        disablePasswordAuthentication: true
+        ssh: {
+          publicKeys: [
+            {
+              path: '/home/${virtualMachines_iagora_name}/.ssh/authorized_keys'
+              keyData: sshPublicKey 
+            }
+          ]
+        }
+        provisionVMAgent: true
+        patchSettings: {
+          patchMode: 'AutomaticByPlatform'
+          automaticByPlatformSettings: {
+            rebootSetting: 'IfRequired'
+            bypassPlatformSafetyChecksOnUserSchedule: false
+          }
+          assessmentMode: 'ImageDefault'
+        }
+      }
+      secrets: []
+      allowExtensionOperations: true
+      //requireGuestProvisionSignal: true
+    }
+    securityProfile: {
+      uefiSettings: {
+        secureBootEnabled: true
+        vTpmEnabled: true
+      }
+      securityType: 'TrustedLaunch'
+    }
+    networkProfile: {
+      networkInterfaces: [
+        {
+          id: networkInterfaces_iagora731_z1_name_resource.id
+          properties: {
+            deleteOption: 'Detach'
+          }
+        }
+      ]
+    }
+    diagnosticsProfile: {
+      bootDiagnostics: {
+        enabled: true
+      }
+    }
+  }
+}
+
+//subnet ressourc
```

Comment:
Should we always use upper-case letters for the title or lower-case letters?

Reply:
lower-case letters

Reply:
Ok) then the following comments, for example:
```
// VIRTUAL NETWORK RESSOURCE
// PUBLIC IP ADDRESS RESSOURCE
```
Would become:
```
// Virtual Network Resource
// Public IP Address Resource
```
Right?